### PR TITLE
Feat: added Pages

### PR DESCRIPTION
**Pull request by Avin008**
### Pages Includes:

- Home-Page
- Archive-Page
- Trash-Page
- index.js

### CODE REVIEW

**Review by TusharJade — APPROVED** [review]
Go ahead, good to go for merge👍

File: src/components/CreateNote/CreateNote.jsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    color: "",
+    priority: "",
+    label: "",
+    isTrashed: false,
+    isArchived: false,
+    deletedFromTrash: false,
+    createdAt: new Date(),
+  });
+
+  console.log(notes);
```
Comment (junaidshaikh-js) [inline src/components/CreateNote/CreateNote.jsx:29]: Remove console logs

File: src/components/EditNote/edit-note.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+select {
+  background-color: transparent;
+  color: var(--color-white);
+}
+
+option {
+  background-color: var(--color-black);
+}
```
Comment (junaidshaikh-js) [inline src/components/EditNote/edit-note.css:94]: CSS properties are repeated, it is better to keep the common CSS in single file.

**Review by junaidshaikh-js — COMMENTED** [review]
Good work. 

Suggested one change. Rest looks good.

**Pull request merged**